RecoLocalFastTime/FTLCommonAlgos/plugins/BTLUncalibRecHitAlgo.cc

```diff
@@ -50,7 +50,7 @@ FTLUncalibratedRecHit BTLUncalibRecHitAlgo::makeRecHit(const BTLDataFrame& dataF
   std::pair<float, float> amplitude(0., 0.);
   std::pair<float, float> time(0., 0.);
 
-  std::pair<float, float> position(-5.f, -5.f);  //position in unit mm
+  float position = -5.f;  //position in cm
```

Comment:
should this be ```-1.f``` to match the defaults set in the ```FTLUncalibratedRecHit```? or does this -5 need to have this special (magic) value?

Reply:
Hi,
I am sorry that I think it should be set to -1.f
Thanks